brokerbot.go

```diff
@@ -104,18 +104,31 @@ func main() {
 		port = "8080"
 	}
 
+	server := &http.Server{
```

Comment:
I know it might be against golang convention to have short variables, but it might be worth for us to name this "httpServer" or something more specific, since I was confused what scope this server var represented

Reply:
Done.